CONTRIBUTORS.md

```diff
@@ -1,3 +1,4 @@
 # Contributors
 
 - collective [ericof@plone.org]
+- macagua, leonardocaballero@gmail.com
```

Comment:
In case you don't want to get spammed, you can also provide your GitHub username only by preceding it with `@`.

Reply:
@stevepiercy Done with commit 15605c8